TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -732,7 +737,13 @@ protected function evaluatePropertyAnnotations(ClassMetadataInfo $metadata)
                             break;
                         default:
                             if (strpos($propertyMetaData['type'], '\\') !== false) {
-                                if ($this->reflectionService->isClassAnnotatedWith($propertyMetaData['type'], Flow\ValueObject::class)) {
+                                if ($valueObjectAnnotation = $this->reflectionService->getClassAnnotation($propertyMetaData['type'], Flow\ValueObject::class)) {
```

Comment:
Yeah, I'll have to recheck, but it didn't work out first time when I simply "continue;"'d, so I just quickly tried the "continue 2;" - might have been some caching weirdness.